testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/AbstractHttpClientTest.java

```diff
@@ -1119,6 +1120,10 @@ SpanDataAssert assertClientSpan(
               if (httpClientAttributes.contains(UrlAttributes.URL_FULL)) {
                 assertThat(attrs).containsEntry(UrlAttributes.URL_FULL, uri.toString());
               }
+              if (httpClientAttributes.contains(UrlIncubatingAttributes.URL_TEMPLATE)) {
+                assertThat(attrs)
+                    .containsEntry(UrlIncubatingAttributes.URL_TEMPLATE, uri.getPath());
```

Comment:
it might be nice to have template be something other than uri path so it doesn't look like that's the intention